docs/modules/ROOT/pages/mutations/create.adoc

```diff
@@ -96,3 +96,48 @@ mutation {
 ----
 
 This will create a User with name "John Doe", an introductory Post, both of which will be returned with their autogenerated IDs.
+
+== `connectOrCreate` relations
+If a related node has a `@unique` directive defined, `connectOrCreate` can be used in a
+nested create to perform a `MERGE` operation on the related node, creating a new
+relation and the related node if it doesn't exists.
```

Comment:
Pointing out redundant new lines, and also the fact that the `@id` directive should also work to enable `connectOrCreate` operations.

```suggestion
== `connectOrCreate` relations

If a related node has at least one `@unique` or `@id` (without `unique` argument set to `false`) directive defined, `connectOrCreate` can be used in a nested create to perform a `MERGE` operation on the related node, creating a new relation and the related node if it doesn't exists.
```

Reply:
Regarding new lines:

These were not produced by my tooling, I wrote it like that just to improve readability as plain text (happy to remove them for consistency). I think there is no need to worry regarding formatting, I already checked and adoc works like markdown, in that newlines are not taken into account when formatting (with the idea of making the text legible in plain text without automatic word wrapping) only multiple new lines are considered as separate paragraphs

Reply:
We need to choose one approach, and that approach should be enforced by automatic linting and formatting. At the moment we have half the team doing one thing and half the other, so let's agree on an approach and find the right tool to do it for us.

Reply:
I was taught that you should format md and asciidoc by having a max line width. One of the reasons should be easier diffing if I remember correctly. And the other one that we don't have to rely on the editors wrapping lines to make it easy to read.

Single newlines has no effect in the output.  
If you actually want a newline in the output you should add two newlines or insert a double space at the end of a line.

Setting up automatic linting and formatting is a great idea, but I would vote on inserting newlines rather than have the editors wrap lines.

Reply:
Happy to run with that, know this was the case for markdown but was unsure on the effects of newlines with AsciiDoc.

It's a draft document, but an interesting recommendation from AsciiDoc here: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line